bsp/esp-box/idf_component.yml

```diff
@@ -1,5 +1,5 @@
 
-version: "3.1.0"
+version: "3.2.0"
```

Comment:
This is actually a breaking change, because from now on, the user's cannot user I2C legacy driver with this BSP.

The whole topic is probably more complicated than we anticipated, we can discuss offline